src/components/integrated/OakLessonReviewItem/OakPupilLessonReviewItem.tsx

```diff
@@ -0,0 +1,174 @@
+      $ph={["inner-padding-m", "inner-padding-xl"]}
+      $pv="inner-padding-l"
+      $borderRadius="border-radius-l"
+      $borderColor={completed ? completedBackgroundColor : borderColor}
+      $ba="border-solid-l"
+      {...rest}
+    >
+      <OakRoundIcon
+        iconName={pickIconForSection(lessonSectionName)}
+        $width="all-spacing-10"
+        $height="all-spacing-10"
+        $background={iconBackgroundColor}
+      />
+      <OakFlex $flexGrow={1} $flexShrink={1} $flexDirection={"column"}>
+        <OakBox $font={["heading-6", "heading-5"]} $color={"text-primary"}>
+          {pickLabelForSection(lessonSectionName)}
+        </OakBox>
+        <OakBox $font={["body-2", "body-1"]}>
+          {pickSummaryForProgress(props)}
+        </OakBox>
+      </OakFlex>
+      {renderQuestionCounter(props)}
+    </StyledLessonReviewItem>
+  );
+};
+
+function pickSummaryForProgress(props: OakLessonReviewItemProps) {
+  switch (props.completed) {
+    case false:
+      return pickSummaryForIncomplete(props);
+    case true:
+      return (
+        <OakFlex $gap="space-between-sssx" $alignItems="center">
+          <OakIcon
+            iconName="tick"
+            $width="all-spacing-6"
+            $height="all-spacing-6"
+          />
+          Completed
+        </OakFlex>
+      );
+  }
+}
+
+function pickSummaryForIncomplete(props: OakLessonReviewItemProps) {
+  switch (props.lessonSectionName) {
+    case "intro":
+      return "Prepare";
+    case "starter-quiz":
+      return `Activate - ${props.numQuestions} questions`;
+    case "exit-quiz":
+      return `Check ${props.numQuestions} questions`;
+    case "video":
+      return `Learn`;
+  }
+}
+
+function pickLabelForSection(sectionName: LessonSectionName): string {
+  switch (sectionName) {
+    case "intro":
+      return "Intro";
+    case "starter-quiz":
+      return "Starter quiz";
+    case "video":
+      return "Watch video";
+    case "exit-quiz":
+      return "Exit quiz";
+  }
+}
+
+function pickColorsForSection(
+  sectionName: LessonSectionName,
+): [
+  completedBackgroundColor: OakCombinedColorToken,
+  borderColor: OakCombinedColorToken,
+  iconBackgroundColor: OakCombinedColorToken,
+] {
+  switch (sectionName) {
+    case "intro":
+      return ["bg-decorative2-very-subdued", "border-decorative2", "aqua"];
+    case "starter-quiz":
+      return ["bg-decorative1-very-subdued", "border-decorative1", "mint"];
+    case "video":
+      return ["bg-decorative4-very-subdued", "border-decorative4", "pink"];
+    case "exit-quiz":
+      return ["bg-decorative5-very-subdued", "border-decorative5", "lemon"];
+  }
+}
+
+function pickIconForSection(sectionName: LessonSectionName): OakIconName {
+  switch (sectionName) {
+    case "intro":
+      return "intro";
+    case "starter-quiz":
+    case "exit-quiz":
+      return "quiz";
+    case "video":
+      return "video";
+  }
+}
+
+function renderQuestionCounter(props: OakLessonReviewItemProps) {
```

Comment:
One more code style thing. I prefer arrow functions whenever possible. I meant to enforce this in the linter but never got round to it. I don't want these to creep in too much.